content/product/discover/users.md

```diff
@@ -64,11 +64,11 @@ Your partners likely have an incomplete view of the current state or what their
 
 ## Resources
 
-- [How to conduct user research [1]](https://guides.18f.gov/methods/): 18F Methods provide instruction on several different research activities to consider, including [interviews](https://guides.18f.gov/methods/stakeholder-and-user-interviews), [contextual inquiries](https://guides.18f.gov/methods/contextual-inquiry), [heuristic evaluation](https://guides.18f.gov/methods/heuristic-evaluation), and more.
+- [How to conduct user research [1]](https://guides.18f.gov/methods/discover): 18F Methods provide instruction on several different research activities to consider, including [interviews](https://guides.18f.gov/methods/discover/stakeholder-and-user-interviews), [contextual inquiries](https://guides.18f.gov/methods/discover/contextual-inquiry), [heuristic evaluation](https://guides.18f.gov/methods/discover/heuristic-evaluation), and more.
```

Comment:
It looks like these links got missed during the migration, because:
- they're using hardcoded "https://guides.18f.gov" as the root, which doesn't work for local development or (I suspect at least) internal link checking
- they're not using the [11ty URL pattern](https://github.com/18F/guides/blob/main/docs/development.md#base-urls).

**change**: Change all the links here to use the [11ty URL pattern](https://github.com/18F/guides/blob/main/docs/development.md#base-urls).

The first link here would be:

```md
[How to conduct user research [1]]({{ "/methods/discover" | url }}): 18F Methods ... # and so on
```